### PR TITLE
Add error pages to orchestra project

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

**Comment by joshblum** [comment]
screen shot 2016-04-09 at 12 37 20

File: orchestra/static/dist/orchestra/common/css/orchestra.css [hunk 1]
```diff
@@ -587,3 +587,27 @@ a.logo {
     cursor: pointer; }
     .project-management .project-actions .project-actions-menu {
       font-size: 2em; }
+
+.error-page {
+  text-align: center;
+  font-family: 'Avenir Next', Helvetica, sans-serif;
```
Comment (marcua) [inline orchestra/static/dist/orchestra/common/css/orchestra.css]: You should use the Orchestra fonts, etc.

**Comment by marcua** [comment]
lgtm with a few fixes

**Comment by joshblum** [comment]
updated look:

screen shot 2016-04-11 at 13 54 31

**Pull request merged**